SoftLayer/managers/load_balancer.py

```diff
@@ -100,7 +100,20 @@ def get_lbaas_uuid_id(self, identifier):
             this_lb = self.lbaas.getLoadBalancer(identifier, mask="mask[id,uuid]")
         else:
             this_lb = self.lbaas.getObject(id=identifier, mask="mask[id,uuid]")
-        return this_lb['uuid'], this_lb['id']
+        return this_lb.get('uuid'), this_lb.get('id')
+
+    def get_lbaas_by_address(self, address):
+        """Gets a LBaaS by address.
+
+        :param address: Address of the LBaaS instance
+        """
+        this_lb = {}
+        this_lbs = self.lbaas.getAllObjects()
+        for lbaas in this_lbs:
+            if lbaas.get('address') == address:
```

Comment:
Is it possible to lookup the lb by the address using an object filter instead?

Reply:
An object filter was the first option that I tried, the object filter works with id, uuid, and name properties, but for some reason when address property is used , it doesn't work.

Reply:
apparently `address` isn't a "real" property. I don't want to use `getAllObjects` every time we look up an id.